waku/v2/peermanager/mock_peer_manager.go

```diff
@@ -0,0 +1,50 @@
+package peermanager
```

Comment:
The file name should be mock_peer_discoverer as it is actually mocking discovery.